src/component/parser/json/converter/utils.ts

```diff
@@ -45,6 +45,9 @@ export class ConvertedElements {
   findMessageFlow(id: string): MessageFlow {
     return this.messageFlows.get(id);
   }
+  getMessageFlows(): MessageFlow[] {
```

Comment:
**thought:** during discussion with @csouchet, we mentioned we could have a single `getFlows` method here to simplify the code in the `ProcessConverter`. This is the only place where we need them, so there is no need to get flows by kind.